<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CSimulatorSelector</class>
 <widget class="QFrame" name="CSimulatorSelector">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>170</width>
    <height>36</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Frame</string>
  </property>
  <property name="frameShape">
   <enum>QFrame::StyledPanel</enum>
  </property>
  <property name="frameShadow">
   <enum>QFrame::Raised</enum>
  </property>
  <layout class="QVBoxLayout" name="vl_SimulatorSelector">
   <property name="spacing">
    <number>0</number>
   </property>
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="QWidget" name="wi_CheckBoxes" native="true">
     <layout class="QHBoxLayout" name="hl_CheckBoxes">
      <property name="spacing">
       <number>3</number>
      </property>
      <property name="leftMargin">
       <number>3</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <item>
       <widget class="QCheckBox" name="cb_FS9">
        <property name="text">
         <string>FS9</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="cb_FSX">
        <property name="text">
         <string>FSX</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="cb_P3D">
        <property name="text">
         <string>P3D</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="cb_XPlane">
        <property name="toolTip">
         <string>XPlane</string>
        </property>
        <property name="text">
         <string>XP</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QWidget" name="wi_RadioButtons" native="true">
     <layout class="QHBoxLayout" name="hl_RadioButtons">
      <property name="spacing">
       <number>3</number>
      </property>
      <property name="leftMargin">
       <number>3</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <item>
       <widget class="QRadioButton" name="rb_FS9">
        <property name="text">
         <string>FS9</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="rb_FSX">
        <property name="text">
         <string>FSX</string>
        </property>
        <property name="checked">
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="rb_P3D">
        <property name="text">
         <string>P3D</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="rb_XPlane">
        <property name="toolTip">
         <string>XPlane</string>
        </property>
        <property name="text">
         <string>XP</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
